canonical-web-and-design/ubuntu.com
# [Image module] /_docs/strips

## Done

- updated /_docs/strips with image_template module
- removed /_docs/engage, it's been superseded by /_docs/engage_pages and is now redundant

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/_docs/strips
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the images are still there (compare to ubuntu.com/_docs/strips)
